### DISCUSSION TITLE
CRON jobs

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#361] [Ideas]
Hi,

How about a possibility to create CRON jobs? 

Example:
I am thinking to create some kind of top 10 (or whatever) of downloaded files. It is not realistic to calculate that every time when someone wants to see that. Instead I would like a CRON job which does that once in a specific period which I could define in HFS.

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by rejetto** [comment]
it's possible with javascript in a plugin, or directly admin/options/server_code
using for example function [setInterval](https://www.google.com/search?q=setinterval)
